packages/asc-ui/src/components/TopTaskButton/TopTaskButtonStyle.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from '@datapunt/asc-core'
+import { themeSpacing, themeColor } from '../../utils'
+
+import IconStyle from '../Icon/IconStyle'
+
+export interface Props {
+  icon: React.ElementType
+  title?: string
+  content?: string
+  onDarkBackground?: boolean
+  width: string
+  className: string
```

Comment:
This is already typed when you type the React component in the tsx file (see previous comment)

Reply:
done